base/remote/R/remote.copy.to.R

```diff
@@ -22,6 +22,18 @@ remote.copy.to <- function(host, src, dst, options = NULL, delete = FALSE, stder
   if (as.logical(delete)) {
     args <- c(args, "--delete")
   }
+  if (is.null(host)) {
+    PEcAn.logger::logger.severe("`host` object passed to the function is NULL : Try passing a valid host object")
+  }
+  else {
+    if (is.null(host$name)) {
+      PEcAn.logger::logger.severe("`name` parameter in the `host` object is NULL : Try passing a valid host object")
+    } else if (is.null(host$user)) {
```

Comment:
Are name and user both required if tunnel exists? It's been a while since I've used this, but my recollection was that if you had a valid tunnel you didn't need the user?